packages/utils/src/instrument.ts

```diff
@@ -116,13 +116,13 @@ function instrumentConsole(): void {
       return;
     }
 
-    fill(global.console, level, function (originalConsoleLevel: () => any): Function {
+    fill(global.console, level, function (originalConsoleMethod: () => any): Function {
       return function (...args: any[]): void {
         triggerHandlers('console', { args, level });
 
         // this fails for some browsers. :(
-        if (originalConsoleLevel) {
-          Function.prototype.apply.call(originalConsoleLevel, global.console, args);
+        if (originalConsoleMethod) {
+          originalConsoleMethod.call(global.console, args);
```

Comment:
Hi @AbhiPrasad,
Seems like there should be `apply` instead of `call`.
![image](https://user-images.githubusercontent.com/41228762/153840478-c1e31292-74ca-411e-89da-adf110564e3f.png)